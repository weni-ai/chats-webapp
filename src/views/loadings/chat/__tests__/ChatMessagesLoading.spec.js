import { mount } from '@vue/test-utils';
import ChatMessagesLoading from '../ChatMessages.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(ChatMessagesLoading);
};

describe('ChatMessagesLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
