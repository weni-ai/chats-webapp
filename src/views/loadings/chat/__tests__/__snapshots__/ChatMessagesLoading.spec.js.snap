// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ChatHeaderLoading.vue > to match snapshot 1`] = `
"<main data-v-e8870554="" class="chat-messages-loading">
  <div data-v-e8870554="" class="chat-messages-loading__messages"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; width: 80px; height: 23px;"> ‌ </span></span>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
  </div>
</main>"
`;

exports[`ChatMessagesLoading.vue > to match snapshot 1`] = `
"<main data-v-e8870554="" class="chat-messages-loading">
  <div data-v-e8870554="" class="chat-messages-loading__messages"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; width: 80px; height: 23px;"> ‌ </span></span>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
    <div data-v-e8870554="" class="message"><span data-v-e8870554=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>
  </div>
</main>"
`;
